scripts/esbuilder/esbuilder.js

```diff
@@ -9,9 +9,30 @@ const camelCase = (name) => {
 };
 
 const args = process.argv.slice(3).reduce((processArgs, val) => {
-  let [key, value] = val.split('=');
+  let [key, value = true] = val.split('=');
```

Comment:
NOTE: Needed to update esbuilder to account for multiple, arg-provided `--external` arguments.